realtime/redisSubscriber.js

```diff
@@ -21,14 +22,22 @@ const sub = require('../cache/redisCache').client.sub;
  */
 module.exports = (io) => {
   sub.on('message', (channel, mssgStr) => {
-    // message object to be sent to the clients
-    const mssgObj = JSON.parse(mssgStr);
-    const key = Object.keys(mssgObj)[0];
+    zlib.inflate(new Buffer.from(mssgStr, 'base64'),
```

Comment:
.inflate accepts a string input too. However the error `Error inflating! { Error: incorrect header check
    at Zlib._handle.onerror (zlib.js:370:17) errno: -3, code: 'Z_DATA_ERROR' }`
still happens with string input